src/verifiers/csv/csv_utils.c

```diff
@@ -0,0 +1,40 @@
+/* Copyright (c) 2022 Hygon Corporation
+ * Copyright (c) 2020-2022 Alibaba Cloud
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <stdio.h>
```

Comment:
Is `#include <stdio.h>` necessary?

Reply:
It's not necessary, i will delete this line